watertap/unit_models/tests/test_uv_aop.py

```diff
@@ -160,9 +174,9 @@ def test_build(self, UV_frame):
                 assert hasattr(blk[0], obj_str)
 
         # test statistics
-        assert number_variables(m) == 35
-        assert number_total_constraints(m) == 22
-        assert number_unused_variables(m) == 0  # vars from property package parameters
+        assert number_variables(m) == 41
+        assert number_total_constraints(m) == 27
+        assert number_unused_variables(m) == 1  # vars for watertap costing package
```

Comment:
Which unused var is it that gets used for costing? I only ask because the first thought that comes to mind is that maybe that var should only be in the costing package but not sure.

Reply:
The unused variable is `num_of_reactors`, since the costing data I used is for a uv system, so I add this `num_of_reactors` to account for number of reactors in a uv system.

Reply:
Does this represent number of reactors in series, parallel, or both? If parallel, would the var end up being used in the model without costing?

Reply:
It represent reactors in parallel. This var is onlly used for costing the uv reactor.

Reply:
I think we could keep this `num_of_reactors` as a unit model variable because we could use it for the uv dose expression (related to UVT, flow rate and number of banks).